File: CStoreManagement/app/src/main/java/com/c_store/zhiyazhang/cstoremanagement/utils/recycler/onMoveAndSwipedListener.java
package com.c_store.zhiyazhang.cstoremanagement.utils.recycler;

/**
 * Created by zhiya.zhang
 * on 2017/5/26 9:06.
 */

public interface onMoveAndSwipedListener {

    boolean onItemMove(int fromPosition, int toPosition);

    void onItemDismiss(int position);

}

File: CStoreManagement/app/src/main/java/com/cstore/zhiyazhang/cstoremanagement/utils/recycler/onMoveAndSwipedListener.kt
package com.zhiyazhang.mykotlinapplication.utils.recycler

/**
 * Created by zhiya.zhang
 * on 2017/6/2 17:14.
 */
interface onMoveAndSwipedListener {
    fun onItemMove(fromPosition: Int, toPosition: Int): Boolean
    fun onItemDismiss(position: Int)
}